sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Breaking Changes
+- OutputModelUri now returns a uri to a .zip file containing the .ou file instead of the .ou file itself when using the default service version
```

Comment:
```suggestion
- `OutputModelUri` now returns a `Uri` to a `.zip` file containing the `.ou` file instead of the `.ou` file itself when using the new default service version: `V0_3_preview_0`.
```